go-controller/pkg/ovn/namespace.go

```diff
@@ -364,14 +367,11 @@ func (oc *Controller) updateNamespace(old, newer *kapi.Namespace) error {
 				"ACL logging is set to deny=%s allow=%s, err: %q",
 				newer.Name, nsInfo.aclLogging.Deny, nsInfo.aclLogging.Allow, err)
 		}
-		if len(nsInfo.networkPolicies) > 0 {
-			// deny rules are all one per namespace
-			if err := oc.setNetworkPolicyACLLoggingForNamespace(old.Name, nsInfo); err != nil {
-				errors = append(errors, err)
-			} else {
-				klog.Infof("Namespace %s: NetworkPolicy ACL logging setting updated to deny=%s allow=%s",
-					old.Name, nsInfo.aclLogging.Deny, nsInfo.aclLogging.Allow)
-			}
+		if err := oc.handleNetPolNamespaceUpdate(old.Name, nsInfo); err != nil {
```

Comment:
This is handled by `sharedNetpolPortGroups` now, it won't update port group that doesn't exist
But I updated the logic to return nil instead of an error in that case

Reply:
ok cool I see:
```
		_, loaded := oc.sharedNetpolPortGroups.Load(pgKey)
		if !loaded {
			// shared port group doesn't exist, nothing to update
			return nil
		}
```